src/app/suppliers/why-join/page.tsx

```
@@ -0,0 +1,281 @@
+import { ArrowRight, Users, Eye, Clock, Zap, Sparkle, Sparkles, MapPin, Infinity, HeartHandshake } from 'lucide-react'
```

Biome (2.1.2)

[error] Do not shadow the global "Infinity" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)